docker-compose.test.yaml

```diff
@@ -4,3 +4,4 @@ services:
     app:
         build:
             target: test
+        image: liberotest/dummy-api:${IMAGE_TAG:-master}
```

Comment:
We won't need this? The `prod` image will be the one pushed.

Reply:
We don't need to push this one, but avoids the `smoke` and `phpunit`/`phpcs`/`phpstan` phases overwriting the same image due to different `docker-compose` configurations being used

Reply:
Ok; might be better to avoid the fake namespace (so `libero/test-dummy-api:${IMAGE_TAG:-master}` or `libero/dummy-api:${IMAGE_TAG:-master}-test`?).

Reply:
Yes, `libero/.*` should be mantained as a standard